docker-compose.yml

```diff
@@ -1,19 +1,24 @@
 version: '3'
 services:
   eddi:
-    image: labsai/eddi:latest
+    image: labsai/eddi:4.8.0-kkorsakov
```

Comment:
this needs to be reverted back to 'latest'
```suggestion
    image: labsai/eddi:latest
```

Reply:
> this needs to be reverted back to 'latest'

Reverted, thanks.